pkgs/os-specific/darwin/apple-source-releases/libffi/package.nix

```diff
@@ -50,9 +50,15 @@ mkAppleDerivation (finalAttrs: {
     "--with-gcc-arch=generic" # no detection of -march= or -mtune=
   ];
 
-  # Make sure aarch64-darwin is using the trampoline dylib.
-  postConfigure = lib.optionalString stdenv.hostPlatform.isAarch64 ''
-    echo '#define FFI_TRAMPOLINE_WHOLE_DYLIB 1' >> aarch64-apple-darwin/fficonfig.h
+  postConfigure = ''
+    # Use Apple’s configuration instead of the one generated by the `configure` script.
+    cp darwin/include/fficonfig_${stdenv.hostPlatform.darwinArch}.h ${stdenv.hostPlatform.config}/fficonfig.h
```

Comment:
Is the missing `include/` here correct? I guess so, going by https://github.com/apple-oss-distributions/libffi/blob/83edf1120fbb1d9aace9b9eb87ac22198b1bc2e8/darwin/README?

Also, does `tramp.h` matter? Do we need to move that into place too?

We could potentially run https://github.com/apple-oss-distributions/libffi/blob/83edf1120fbb1d9aace9b9eb87ac22198b1bc2e8/generate-darwin-source-and-headers.py ourselves if it’d make anything easier/cleaner…

Reply:
Thanks. This all seems reasonable to me then.